apps/frontend/static_src/scss/components/footer.scss

```diff
@@ -23,6 +31,7 @@
 
     &__legal {
         @include fs(xxs);
+        text-align: center;
```

Comment:
This is also applied on desktop.

<img width="1800" alt="Screenshot 2022-11-04 at 15 09 16" src="https://user-images.githubusercontent.com/1969342/199997344-b986611d-ab44-41f8-8a61-d964fb4539ce.png">